py/expression.cpp

```diff
@@ -141,12 +141,13 @@ Expression_mul( PyObject* first, PyObject* second )
     return BinaryInvoke<BinaryMul, Expression>()( first, second );
 }
 
-
-static PyObject*
-Expression_div( PyObject* first, PyObject* second )
-{
-    return BinaryInvoke<BinaryDiv, Expression>()( first, second );
-}
+#if PY_MAJOR_VERSION < 3
+    static PyObject*
+    Expression_div( PyObject* first, PyObject* second )
+    {
+        return BinaryInvoke<BinaryDiv, Expression>()( first, second );
+    }
+#endif
```

Comment:
2.7: https://docs.python.org/2.7/c-api/typeobj.html#number-object-structures
3.4: https://docs.python.org/3.4/c-api/typeobj.html#number-object-structures

`nb_divide` is gone in Py3k. Should it be moved to `nb_true_divide`? `nb_floor_divide`? Both?